Prox/Prox/Utilities/PlaceUtilities.swift

```diff
@@ -21,6 +22,43 @@ struct PlaceUtilities {
         }
     }
 
+    static func sort(places: [Place], byTravelTimeFromLocation location: CLLocation, ascending: Bool = true, completion: @escaping ([Place]) -> ()) {
+        PlaceUtilities.getTravelTimes(forPlaces: places, fromLocation: location).upon { result in
+            let sortedPlaces = places.sorted { (placeA, placeB) -> Bool in
+                guard let placeATravelTimes = PlaceUtilities.lastTravelTimes(forPlace: placeA),
+                    let placeBTravelTimes = PlaceUtilities.lastTravelTimes(forPlace: placeB) else {
+                        return false
```

Comment:
This logic is inconsistent – we'd put a null A after B but not put a null B before an A. Where do we want to place items without travel times? In the back? Also, consider what we should do with ascending in this case.

If we want them in the back, maybe we should:
```swift
let placeATravelsTimes = ...
...
let placeAETA = placeATravelTimes?.getShortestTravelTime() ?? MAX_VALUE
```

Keep in mind we'd have to rotate for ascending.